anndata/tests/test_transpose.py

```diff
@@ -25,15 +25,18 @@ def _add_raw(adata, *, var_subset=slice(None)):
     return new
 
 
-# Cases to add:
+# TODO: Cases to add:
 # * Views
-# * X is None
+# * X is None should have the xfail marker removed
 # * Backed
 @pytest.fixture(
     params=[
         pytest.param(gen_adata((50, 20)), id="csr_X"),
         pytest.param(gen_adata((50, 20), sparse.csc_matrix), id="csc_X"),
         pytest.param(_add_raw(gen_adata((50, 20))), id="with_raw"),
+        pytest.param(
+            gen_adata((20, 10), X_type=None), id="None_X", marks=pytest.mark.xfail
+        ),
```

Comment:
@rcannood, this should suffice for a transpose test. For getting transpose to work with `X=None`, I'd remove the `marks` from this case, and try to get this to pass.

Reply:
Done in #463.